home-api-tests/pom.xml

```diff
@@ -2,15 +2,313 @@
+                                                </args>
+                                            </exec>
+                                        </entryPoint>
+                                        <assembly>
+                                            <basedir>/</basedir>
+                                            <inline>
+                                                <files>
+                                                    <file>
+                                                        <source>
+                                                            ${docker.artifacts.path}/${home.application.jar.name}-${home.application.jar.version}.jar
+                                                        </source>
+                                                        <outputDirectory>/</outputDirectory>
+                                                    </file>
+                                                </files>
+                                            </inline>
+                                        </assembly>
+                                    </build>
+                                    <run>
+                                        <ports>
+                                            <port>${home.application.external.port}:${home.application.internal.port}
+                                            </port>
+                                        </ports>
+                                        <links>
+                                            <link>${postgres.container.name}</link>
+                                        </links>
+                                        <wait>
+                                            <log>Started HomeApplication</log>
+                                            <time>30000</time>
+                                        </wait>
+                                    </run>
+                                </image>
+                            </images>
+                        </configuration>
+                        <executions>
+                            <execution>
+                                <id>start</id>
+                                <phase>pre-integration-test</phase>
+                                <goals>
+                                    <goal>stop</goal>
+                                    <goal>build</goal>
+                                    <goal>start</goal>
+                                </goals>
+                            </execution>
+                            <execution>
+                                <id>stop</id>
+                                <phase>post-integration-test</phase>
+                                <goals>
+                                    <goal>stop</goal>
+                                </goals>
+                            </execution>
+                        </executions>
+                    </plugin>
+                    <plugin>
```

Comment:
Please move this definition to the `<id>api-tests</id>` profile

Reply:
Done